frontend/src/app/components/wp-relations/wp-relations-create/wp-relations-autocomplete/wp-relations-autocomplete.component.ts

```diff
@@ -129,49 +152,22 @@ export class WorkPackageRelationsAutocomplete implements AfterContentInit {
         .ensureLoaded(workPackage)
         .then(() => {
           this.onSelected.emit(workPackage);
-          this.ngSelectComponent.close();
+          this.ngSelectComponent.ngSelectInstance.close();
         });
     }
   }
 
-  private autocompleteWorkPackages(query:string):Observable<WorkPackageResource[]> {
-    // Return when the search string is empty
-    if (query === null || query.length === 0) {
-      this.isLoading = false;
-      return of([]);
-    }
-
-    // Remove prefix # from search
-    query = query.replace(/^#/, '');
-
-    return from(
-      this.workPackage.availableRelationCandidates.$link.$fetch({
-        query: query,
-        filters: JSON.stringify(this.additionalFilters),
-        type: this.filterCandidatesFor || this.selectedRelationType
-      }) as Promise<WorkPackageCollectionResource>
-    )
-      .pipe(
-        map(collection => collection.elements),
-        catchError((error:unknown) => {
-          this.notificationService.handleRawError(error);
-          return of([]);
-        }),
-        tap(() => this.isLoading = false)
-      );
-  }
-
-  onOpen() {
+    onOpen() {
     // Force reposition as a workaround for BUG
     // https://github.com/ng-select/ng-select/issues/1259
     setTimeout(() => {
-      const component = (this.ngSelectComponent) as any;
+      const component = (this.ngSelectComponent.ngSelectInstance) as any;
```

Comment:
The same, this should be inside the OpAutocompleterComponent and inside ngZone.runOutsideAngular. This way the applications will end up polluted with a ton of setTimeouts only for opening and closing the OpAutocompleterComponent.